Form-Controls/index.html

```diff
@@ -14,14 +14,50 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div>
+      <label for="name">Name</label>
+      <input type="text" name="name" id="name" required minlength="2" pattern="[A-Za-z\s]+" title="Name must be at least 2 characters and can only contain letters and spaces.">
```

Comment:
Interesting! What would happen if they were called O'Donnell?

(This criteria is set just so you can think about this: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)